src/WebUI/Components/KubeSummary.tsx

```diff
@@ -175,6 +187,15 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
                     key={format("dc-{0}", this.state.namespace || "")}
                     onItemInvoked={this._onDeploymentItemInvoked}
                 />
+                <DaemonSetListingComponent

```

Comment:
DaemonSetListingComponent and stateful should ideally go to deployments component, so we have a clean kubesummary? we can rename the deploymentscomponent if needed